app/pages/lab/workflow-viewer/task-helpers.cjsx

```diff
@@ -0,0 +1,58 @@
+React = require 'react'
+TriggeredModalForm = require 'modal-form/triggered'
+tasks = require '../../../classifier/tasks'
+{Markdown} = require 'markdownz'
+
+# All classes in this file take in `workflow` and `taskKey` as props
+
+# Render the task name box
+TaskName = React.createClass
+  displayName: 'TaskName'
+
+  isEmptyStr: (str) ->
+    str.replace(/^\s+|\s+$/g, '').length == 0
+
+  render: ->
+    task = @props.wf[@props.taskKey]
+    task_text = tasks[task.type].getTaskText(task)
+    style =
+      'zindex': 'inherit'
+    <div className='answer-item' id={@props.plumbId + '_name'} style={style}>
+      <Markdown className='lab markdown-vis'>{"#{task_text}"}</Markdown>
```

Comment:
Should be able to just do `<Markdown className='lab markdown-vis'>{task_text}</Markdown>` yea?